staging/src/k8s.io/apiextensions-apiserver/test/integration/apiapproval_test.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Copyright 2020 Authors of Arktos - file modified.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// File modified by cherrypick from kubernetes on 02/25/2021
+package integration
+
+import (
+	"testing"
+	"time"
+
+	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
```

Comment:
the test is for v1beta1, instead of v1?

Reply:
This is added as part of reserved apigroups which is before promote to CRD to V1. and we can still use v1beta1.